src/gwc/src/test/java/org/geoserver/gwc/GWCSeedingSecurityIntegrationTest.java

```diff
@@ -120,7 +120,7 @@ public void testSeedPublic() throws GeoWebCacheException, InterruptedException,
         assertNull("Cache should be empty at test start", tileObject.getBlob());
 
         SeedRequest sr = createSeedRequest(PUB_LAKES);
-        @SuppressWarnings("PMD.UnusedLocalVariable")
+        @SuppressWarnings({"PMD.UnusedLocalVariable", "unused"})
```

Comment:
I don't see an "unused" warning in the Java compiler lint settings... is this Eclipse specific?